actionlist/app/actions/action_items.js

```diff
@@ -1,8 +1,8 @@
 import { createAction } from 'redux-actions';
-import { TOGGLE_DRAWER } from './types';
+import { TOGGLE_DRAWER, SORT_BY_CREATION_DATE, SORT_BY_LAST_APPROVED_DATE, SORT_BY_PROCESS_TYPE, SORT_BY_ACTION_REQUIRED } from './types';
 
-/* eslint-disable import/prefer-default-export */
-/* Needs to be done as actions are not dispatched without it.
-Will be removed when we have multiple actions */
 export const toggleDrawer = createAction(TOGGLE_DRAWER);
-/* eslint-enable import/prefer-default-export */
+export const sortByCreationDate = createAction(SORT_BY_CREATION_DATE);
```

Comment:
Instead of having multiple sorting action types, why not have a `sort` action that takes a field name, a value, and an order (ascending or descending) as the payload?